server/bin/pbench-index.py

```diff
@@ -6,30 +6,39 @@
 
 """
 
-import sys
-import os
 import glob
+import hashlib
+import json
+import lzma
+import math
+import os
+import socket
+import sys
 import tarfile
 import tempfile
 from argparse import ArgumentParser
 from configparser import Error as ConfigParserError
+from configparser import Error as NoSectionError
+from configparser import NoOptionError
```

Comment:
Consider combining these into a single statement, like
```
from configparser import (
    Error as ConfigParserError,
    Error as NoSectionError,
    NoOptionError
    )
```

Also, do you _really_ want to be aliasing `Error` **twice** to two different symbols...ones which I _think_ we can import directly from `configparser`?  (That is, I understand and support importing `configparser.Error` as `ConfigParserError`, but I don't think we should **_also_** be importing it as `NoSectionError` -- I think we should just be importing `NoSectionError`.)

Reply:
Fixed.